canonical-web-and-design/charmhub.io
# Add husky and lint-staged

## Done

- Add husky and lint-staged

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run it locally, make some changes and then commit. Note husky does the job and runs the linters
